### DISCUSSION TITLE
Team Bounty Quests

### DISCUSSION THREAD

**Discussion by korigard** [zebscripts/AFK-Daily#110] [Ideas]
can make the script check for event up and avoid or not to send the quest? (option on config.ini)

**Comment by Zebiano** [comment]
Can you explain this a little further? I didn't understand it. You're talking about team bounties, found in the bounty board in the dark forest right? And which event are you talking about?

**Reply by korigard** [reply]
https://i.imgur.com/A4gakFz.png this event. i want to refresh my quest for get more diamond during the event. The best will be that during the event in game the script will skip the bounty quest. Sorry for my bad english :P

**Reply by Zebiano** [reply]
> Sorry for my bad english :P

No need to apologise, it's definitely good enough for us to communicate! :)

And I think I now understand what you want. Whenever the event "Bountiful Bounties" event is active, you'd like the script to not auto-run solo bounties, as you get double the rewards there and you only want to run a certain type of bounties to get specific rewards.

I see. It is possible to implement this, though it is very difficult to check which events are currently active. Because of this, and taking into consideration the amount of effort it would take to code this versus the reward it gives to users, I don't think it's worth it to implement such a feature. I'd rather ask you to set `doSoloBounties` to `false` in the `config.ini` while the event is active.

Sorry, and I hope this answer helps!

**Reply by korigard** [reply]
> I see. It is possible to implement this, though it is very difficult to check which events are currently active. Because of this, and taking into consideration the amount of effort it would take to code this versus the reward it gives to users, I don't think it's worth it to implement such a feature. I'd rather ask you to set `doSoloBounties` to `false` in the `config.ini` while the event is active.
> 

set now doSoloBounties to false... too late for today but ok xD
